cover-icon-element.js

```diff
@@ -2,31 +2,34 @@ class CoverIconElement extends HTMLElement {
 
   defaults = {
     breakpoints: [1, 50, 75, 100],
-    color: "#4d4d4d"
+    color: "#4d4d4d",
+    opacity: '0.3'
```

Comment:
The docs require a float for opacity. Specifying a number here too makes it more consistent.
```suggestion
    opacity: 0.3
```

Reply:
you're right, I added the quotes in one of my tests and forgot to remove it